components/mixins/experience-event/industry-verticals/experienceevent-upgrade-details.schema.json

```diff
@@ -31,6 +31,11 @@
         "xdm:upgrade": {
           "type": "object",
           "properties": {
+            "xdm:upgradeSteps": {
+              "title": "Upgrade Steps",
+              "$ref": "https://ns.adobe.com/xdm/datatypes/transaction",
```

Comment:
Should this be "tool-usage" ? majority of step properties are defined as "tool-usage"

Reply:
You are right.  I just pushed a fix.